tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -284,6 +285,12 @@ def test_nodereplacement_proactive(self):
             timeout=1800
         ), "Data re-balance failed to complete"
 
+        ocs_version_semantic = version.get_semantic_ocs_version_from_config()
+        if ocs_version_semantic >= version.VERSION_4_13:
+            assert (
+                verify_only_ocs_nodes_in_storagecluster
```

Comment:
Do we need to add these verification to these node replacement tests as well? test_nodereplacement_proactive_with_io_running and test_nodereplacement_twice

Reply:
done